### PR TITLE
Support the terraform wrapper

### PR DESCRIPTION
**Pull request by ResmiArjun**

### CODE REVIEW

File: submitter/adaptors/terraform_adaptor.py [hunk 1]
```diff
@@ -120,12 +124,15 @@ def __init__(self, adaptor_id, config, dryrun, validate=False, template=None):
         self.vars_file_tmp = "{}terraform.tfvars.json.tmp".format(self.volume)
         self.account_file = "{}accounts.json".format(self.volume)
         self.oci_auth_key = "{}oci_api_key.pem".format(self.volume)
-        self.configure_file = "{}configure".format(self.volume)
+        self.configure_file = "{}configure.py".format(self.volume)
         self.token_file = "{}pyGetScopedToken.py".format(self.volume)
+        self.preprocess_file = "{}preprocess.py".format(self.volume)
```
Comment (jaydesl) [inline submitter/adaptors/terraform_adaptor.py:129]: So each time a new pre-processing script is added to MiCADO, we have to change the Terraform Adaptor as well? Does that not defeat the purpose of having  the wrapper?

File: submitter/adaptors/terraform_adaptor.py [hunk 2]
```diff
@@ -1155,5 +1171,3 @@ def _terraform_customise(self):
         self._terraform_exec(command)
         command = ["sh", "-c", "apk -e info py3-pip || apk add py3-pip && pip3 install PyJWT" + LOG_SUFFIX]
```
Comment (jaydesl) [inline submitter/adaptors/terraform_adaptor.py:1172]: Isn't this now handled with the PostStart hook in [micado-manifest.yml](https://github.com/micado-scale/ansible-micado/blob/develop/roles/micado_master/start/templates/micado/micado-manifest.yml.j2#L305) ?

Comment (ResmiArjun) [reply submitter/adaptors/terraform_adaptor.py:1172]: I think we can remove that terraform_customise function now. I forgot to remove it.

**Review by jaydesl — CHANGES_REQUESTED** [review]
I think the name of the PR should be a bit more descriptive

File: submitter/adaptors/terraform_adaptor.py (hunk 1 above)
Comment (ResmiArjun) [reply submitter/adaptors/terraform_adaptor.py:129]: Currently from what I have done this name will be same for whichever cloud provider you are using. Just that the contents will be different to suit each cloud provider.

**Pull request merged**